### ISSUE TITLE
The accuracy of the trained model by own dataset is bad.

### ISSUE THREAD

**Issue by jiangfan1992** [TJCVRS/CRNN_Tensorflow#14]
dear author,
I used your code in my dataset. In the training stage, the loss value keeps decreasing and the precision keeps rising. But, when I used tools/demo_shadownet.py to test picture, the result is so bad. 
Could you give me some suggestions?
Some information: 
1) My dataset is OCR, consists of 10000 pictures.
2) Base learning rate is 0.1, learning rate step is 30k, epoch is 100k, batchsize is 32.

**Comment by TJCVRS** [comment]
@jiangfan1992 Maybe the model is overfitted on your dataset you can enlarge your dataset to improve the model's performance
